prov/util/src/util_coll.c

```diff
@@ -1033,18 +1033,16 @@ int ofi_join_collective(struct fid_ep *ep, fi_addr_t coll_addr,
 
 	join_op->data.join.new_mc = new_coll_mc;
 
-	if (new_coll_mc->local_rank != FI_ADDR_NOTAVAIL) {
-		ret = ofi_bitmask_create(&join_op->data.join.data, OFI_MAX_GROUP_ID);
-		if (ret)
-			goto err2;
+	ret = ofi_bitmask_create(&join_op->data.join.data, OFI_MAX_GROUP_ID);
+	if (ret)
+		goto err2;
 
-		ret = ofi_bitmask_create(&join_op->data.join.tmp, OFI_MAX_GROUP_ID);
-		if (ret)
-			goto err3;
+	ret = ofi_bitmask_create(&join_op->data.join.tmp, OFI_MAX_GROUP_ID);
+	if (ret)
+		goto err3;
 
-	} else {
+	if (new_coll_mc->local_rank == FI_ADDR_NOTAVAIL)
```

Comment:
Please update commit message to:
```
util/coll: <Describe what is being fixed>
```
and update the message to indicate why the change is being made, because i'm not following it from the commit message.

Reply:
fixed